src/ophys_etl/qc/roi_qc.py

```diff
@@ -1,22 +1,22 @@
 from pathlib import Path
 from timeit import default_timer as timer
 import re
-
 import argschema
 import h5py
 import matplotlib
 import marshmallow as mm
 import numpy as np
-from matplotlib import pyplot as plt
-from matplotlib.backends.backend_pdf import PdfPages
-from scipy.sparse import coo_matrix
-
-from ophys_etl.transforms.roi_transforms import (
+import matplotlib as mpl
+mpl.use('Agg')
+from matplotlib import pyplot as plt  # noqa: E402
```

Comment:
Can you move these guys up (to get rid of the flake8 issue) or do any of them import matplotlib?

Reply:
done.